epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModelGroup.java

```diff
@@ -110,9 +110,10 @@ private EpoxyModelGroup(@LayoutRes int layoutRes, List<EpoxyModel<?>> models) {
   /**
    * Constructor use for DSL
    */
-  protected EpoxyModelGroup() {
+  protected EpoxyModelGroup(@LayoutRes int layoutRes) {
```

Comment:
isn't this breaking existing usages by removing the empty constructor? should this be a new, separate constructor from the empty one?

Reply:
The empty constructor was introduced in #1012, so it was not released yet and couldn't be used by library users. However, I've restored it to not introduce confusion that you have to provide some layout resource even if it's not necessary at this point.